lib/WOPI/Parser.php

```diff
@@ -56,6 +64,7 @@ public function getUrlSrc($mimetype) {
 			];
 		}
 
+		$this->logger->warning('Didn\'t find urlsrc for mimetype {mimetype} in this WOPI discovery response: {discovery}', ['mimetype' => $mimetype, 'discovery' => $discovery]);
```

Comment:
```suggestion
		$this->logger->error('Didn\'t find urlsrc for mimetype {mimetype} in this WOPI discovery response: {discovery}', ['mimetype' => $mimetype, 'discovery' => $discovery]);
```

I think this could even be logged at error level to be sure to catch it in any case.